### ISSUE TITLE
Can't use gradlew

### ISSUE THREAD

**Issue by eygraber** [javalite/activejdbc-gradle#3]
If I try using gradlew I get:

```
4:32:04 PM: Executing external tasks 'clean build run'...
:clean
warning: [options] bootstrap class path not set in conjunction with -source 1.7
1 warning
:compileJava
:instrumentModels
**************************** START INSTRUMENTATION ****************************
Directory: /Users/bobthebuilder/workspace/project1/build/classes/main
:instrumentModels FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':instrumentModels'.
> org.javalite.instrumentation.InstrumentationException: java.lang.RuntimeException: com.java.db.Something class is frozen

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 0.355 secs
com.java.db.Something class is frozen
4:32:05 PM: External tasks execution finished 'clean build run'.
```

**Comment by ipolevoy** [comment]
@jasoma, have an idea? Usually when a class ifs frozen during instrumentation, it is probably because it was already instrumented.

**Comment by jasoma** [comment]
@eygraber Are you using the gradle daemon in your build? There is a known issue with the daemon[1] that can cause this error. If your not can you post your code so I can try to reproduce locally?

[1] in that I know it but haven't written it up yet.

**Comment by eygraber** [comment]
@jasoma I am using a daemon. I'll try again, without and report back.

**Comment by eygraber** [comment]
@jasoma I tested again without the daemon, and it seems to be working. Do you have a workaround for getting it to work with the daemon? I'm not sure if there's a way to use gradle in Intellij without the daemon.

**Comment by jasoma** [comment]
@eygraber I also use Intellij and haven't had a problem. I'm pretty sure Intellij will honor whatever is in your user `gradle.properties` file as far as building via the daemon goes? Alternatively you could add the `--no-daemon` parameter in your run configurations.

I don't have a workaround that is of any real use. You can manually kill the daemon process between builds but that's a lot more work then just passing `--no-daemon` to the build.

**Comment by eygraber** [comment]
@jasoma I've always been operating under [this assumption](http://stackoverflow.com/a/25432294/691639), but I explicitly turned off the daemon in `gradle.properties` which still didn't work, and tried passing `--no-daemon` and that caused gradle execution to fail.

**Comment by jasoma** [comment]
@eygraber you are right, i tried myself on the project I though had worked with intellij and it is broken in the same way. I've opened https://github.com/javalite/activejdbc/issues/438 with some more detail on where I think the problem is.

I can't think of any other workaround for builds via the IDE.
